pkg/pod/pod_test.go

```diff
@@ -473,7 +473,6 @@ func TestPodBuild(t *testing.T) {
 						Image:        images.WorkingDirInitImage,
 						Command:      []string{"/ko-app/workingdirinit"},
 						Args:         []string{filepath.Join(pipeline.WorkspaceDir, "test")},
-						WorkingDir:   pipeline.WorkspaceDir,
```

Comment:
No need. The workingDir only enabled creating directories inside the `/workspace` folder. We already account for it. It does not matter what the workingDir is since the init container only handles absolute paths now.